.github/workflows/build.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           - 13
           - 14
           - 15
+          - 16
```

Comment:
```suggestion
          - 16
          - 17-ea
```

Go for gold?

Reply:
Sure!

Reply:
Looks like Kotlin is not ready yet though